source/resiliency.txt

```diff
@@ -12,12 +12,140 @@ Application and Database Resiliency
    :depth: 1
    :class: onecol
 
-Intro statement
+MongoDB |service| is a modern, highly-performant database that is purpose-built 
+to maintain uptime regardless of infrastructure outages, system maintenance and more.
```

Comment:
Just tweaking this slightly since some of the language feels a little market-y, and to lengthen it
```suggestion
MongoDB |service| is a highly-performant database that is designed 
to maintain uptime regardless of infrastructure outages, system maintenance, and more. Use the guidance on this page to plan settings to maximize the resiliency of your application and database.
```